PWGDQ/Core/MCProng.cxx

```diff
@@ -350,6 +350,17 @@ bool MCProng::ComparePDG(int pdg, int prongPDG, bool checkBothCharges, bool excl
         }
       }
       break;
+    case 600:                     // all dielectron from same mother sources
```

Comment:
I think there are small chances we will work with t-quark hadrons in ALICE, but to respect the pythia convention (1,2-light quarks; 3-strange, 4-charm, 5-beauty, 6-top), please use a code starting with 7,8 or better 9, since we have already an example like that.
Also, the comment may be confusing. At this level we compare just single particles, there is no dielectron here. I suggest to use a comment saying something like "group of codes used in the low mass dielectron cocktail"